### PR TITLE
ListingPage

### PR DESCRIPTION
**Pull request by ark2002**
- Added the Listing page Components
- Linked the new components

### CODE REVIEW

File: src/frontend/components/Filter/Filter.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    top: 5rem;
+    left: 0;
+    padding: 3rem;
+    box-shadow: #0000004d 0px 19px 38px, #00000038 0px 15px 12px;
+    align-items: flex-start;
+    gap: 1rem;
+}
+
+.filter__top {
+    width:100%;
```
Comment (MarkVed17) [inline src/frontend/components/Filter/Filter.css:15]: needs code-formatting here.

File: src/frontend/components/ListingPageCard/ListingPageCard.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    color:var(--white-color);
+    background-color: var(--primary-color);
+    border-radius: 5px;
+    padding: 5px 5px 5px 2px;
+    width: fit-content;
+    margin-bottom: 5px;
+    box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
+}
+
+.card__star{
```
Comment (MarkVed17) [inline src/frontend/components/ListingPageCard/ListingPageCard.css:33]: here as well.

File: src/frontend/components/ListingPageCard/ListingPageCard.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    font-size: 1.3rem;
+    padding:2px;
```
Comment (MarkVed17) [inline src/frontend/components/ListingPageCard/ListingPageCard.css:35]: code-formatting here too & many places. Sounds silly but then the code looks clean.

**Review by MarkVed17 — APPROVED** [review]
The classNames could be better to improve code-readability. Rest all works fine!

**Review by ivinayakg — APPROVED** [review]
good component UI, go ahead and merge it

**Pull request merged**